test/integration/targets/git/tasks/main.yml

```diff
@@ -27,7 +27,11 @@
 - include: change-repo-url.yml
 - include: depth.yml
 - include: checkout-new-tag.yml
-- include: tag-verification.yml
+- include: gpg-verification.yml
+  when: >
+    not gpg_version.stderr and
+    gpg_version.stdout and
```

Comment:
There was a gpg_version >= 1.4.16 check in the old version. Is this not needed here?